test/new-e2e/pkg/runner/ci_profile.go

```diff
@@ -63,8 +63,13 @@ func NewCIProfile() (Profile, error) {
 
 	ciEnvironments := strings.Split(environmentsStr, " ")
 
+	outputRoot := os.Getenv("CI_PROJECT_DIR")
+	if outputRoot == "" {
+		return nil, fmt.Errorf("unable to compute output directory, missing variable CI_PROJECT_DIR")
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Since `CI_PROJECT_DIR` is not used everywhere, why failing ? Should you fail instead only for tests that needs it ?

Reply:
> Since CI_PROJECT_DIR is not used everywhere

What do you mean? My thought process was that the CI profile is only used in our CI where the CI_PROJECT_DIR variable should be set, and if it's not then something is probably wrong so we should fail. Is that not the case?

Reply:
What I mean is that not all tests use `CI_PROJECT_DIR`. It is currently used by windows installer tests to export installer logs and possibly other artifacts from hosts under test and store them on the runner host. Since only windows installer tests use it, I would avoid having an error here, on the generic path. 99% of tests today can run without `CI_PROJECT_DIR` defined, so why preventing them to run ?